docs/notebooks/tremor.ipynb

```diff
@@ -0,0 +1,82 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Tremor analysis"
```

Comment:
```suggestion
    "# Tremor analysis"
```

Reply:
Om titels te maken van markdown, kan je hekjes (#) invoegen vóór de tekst. Één hekje is een grote titel, meerdere hekjes maken de titel steeds kleiner.